Doc/library/asyncio-eventloop.rst

```diff
@@ -532,16 +535,17 @@ Creating network servers
 
    Arguments:
 
-   * The *host* parameter can be a string, in that case the TCP server is
-     bound to *host* and *port*. The *host* parameter can also be a sequence
-     of strings and in that case the TCP server is bound to all hosts of the
-     sequence. If *host* is an empty string or ``None``, all interfaces are
-     assumed and a list of multiple sockets will be returned (most likely one
-     for IPv4 and another one for IPv6).
+   * The *host* parameter can be set to several types which determine behavior:
+       - If *host* is a string, the TCP server is bound to *host* and *port*.
+       - if *host* is a sequence of strings, the TCP server is bound to all
+         hosts of the sequence.
+       - If *host* is an empty string or ``None``, all interfaces are
+         assumed and a list of multiple sockets will be returned (most likely
+         one for IPv4 and another one for IPv6).
 
    * *family* can be set to either :data:`socket.AF_INET` or
      :data:`~socket.AF_INET6` to force the socket to use IPv4 or IPv6.
-     If not set it will be determined from host name
+     If not set, it will be determined from host name
```

Comment:
Maybe "If not set, it will be determined" -> "If not set, the family will be determined"?